### PR TITLE
Add PotPlayer

### PR DESCRIPTION
**Pull request by xmha97**

### CODE REVIEW

**Comment by Zero3K** [comment]
Does it work under ReactOS? If so, please reply with a screenshot of it running under ReactOS.

File: potplayer.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[Section]
+Name = PotPlayer
+Version = 1.7.21295
+License = Freeware
+Description = PotPlayer is a multimedia software player.
+Size = 28.3 MiB
+Category = 2
+URLSite = https://potplayer.daum.net/?lang=en
```
Comment (learn-more) [inline potplayer.txt:8]: Why is this a different domain than the download?

File: potplayer.txt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+URLSite = https://potplayer.daum.net/?lang=uk
+
+[Section.0419]
+Description = PotPlayer - это мультимедийный программный проигрыватель.
+URLSite = https://potplayer.daum.net/?lang=ru
+Size = 28,3 МиБ
+
+[Section.0415]
+Description = PotPlayer to multimedialny odtwarzacz oprogramowania.
+URLSite = https://potplayer.daum.net/?lang=pl
```
Comment (katahiromz) [inline potplayer.txt:39]: Insert a newline after here.

**Comment by learn-more** [comment]
The download domain seems like a scam site.

**Comment by Zero3K** [comment]
Its not a scam site.

**Comment by blackcrack** [comment]
https://en.wikipedia.org/wiki/PotPlayer

The Factory Kakao have fusion with The Factory Daum
therewith this link.. so, @learn-more you be wrong and the closing is unjustified.
 this is a legal side .
Potplayer is a good known Player 

best

**Pull request closed without merging**